ci-operator/step-registry/hypershift/aws/create/hypershift-aws-create-chain.yaml

```diff
@@ -26,7 +26,7 @@ chain:
       # have to use a hack like this.
       [[ ${CONTROLPLANE_OPERATOR_IMAGE} = "LATEST" ]] && CONTROLPLANE_OPERATOR_IMAGE="${HYPERSHIFT_RELEASE_LATEST}"
 
-      CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)"
+      CLUSTER_NAME="$(echo -n $PROW_JOB_ID|sha256sum|cut -c-20)-mgmt"
```

Comment:
Now the `-mgmt` suffix is used everywhere, it is only needed for the mgmt cluster, not the nested hostedcluster (i.E `ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yam`l) but not the rest

Reply:
/hold
https://github.com/openshift/release/pull/31713 merged in the meantime, so this isn't needed anymore